README.md

```diff
@@ -2,35 +2,34 @@
 
 A web application for managing an iRODS Zone.
 
-## Installation & Run
+## Setup
 
-### 1. Clone and repository from Github
+### 1. Install Docker and Docker Compose
+Follow instructions in Docker docs to install Docker and Docker Compose for your OS.
+
+### 2. Clone and repository from Github
 Clone the repository from Github.
```

Comment:
"Github" -> "GitHub"

or... remove this line, as it's the same as the section header